include/date_time.h

```diff
@@ -62,13 +86,22 @@ int date_time_uptime_to_unix_time_ms(int64_t *uptime);
  */
 int date_time_now(int64_t *unix_time_ms);
 
+/** @brief Register an event handler for Date time library events.
+ *
+ *  @param evt_handler Event handler. Handler is de-registered if parameter is
+ *                     NULL.
+ */
+void date_time_register_handler(date_time_evt_handler_t evt_handler);
```

Comment:
Should there be a note that there can only be one handler in the system? Users might run into issues with multiple users of the library in the same application.